File: app/src/main/java/com/ashraf/whatsappvpn/ui/OnboardingActivity.java
package com.ashraf.whatsappvpn.ui;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import androidx.appcompat.app.AppCompatActivity;
import com.ashraf.whatsappvpn.R;

public class OnboardingActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_onboarding);

        Button btnAllow = findViewById(R.id.btn_onboarding_allow);
        btnAllow.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(OnboardingActivity.this, DashboardActivity.class);
                startActivity(intent);
                finish();
            }
        });
    }
}

File: app/src/main/java/com/ashraf/whatsappvpn/ui/OnboardingActivity.kt
package com.ashraf.whatsappvpn.ui

import android.content.Intent
import android.os.Bundle
import android.widget.Button
import androidx.appcompat.app.AppCompatActivity
import com.ashraf.whatsappvpn.R

class OnboardingActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_onboarding)

        val btnAllow = findViewById<Button>(R.id.btn_onboarding_allow)
        btnAllow.setOnClickListener {
            val intent = Intent(this, DashboardActivity::class.java)
            startActivity(intent)
            finish()
        }
    }
}
